modules/tests/photons_control_tests/test_clean.py

```diff
@@ -0,0 +1,152 @@
+# coding: spec
+
+from photons_control.clean import CleanConfig, CleanCycle
+
+from photons_app import helpers as hp
+
+from photons_messages import DeviceMessages, LightMessages, DiscoveryMessages
+
+from photons_products import Products
+
+import pytest
+
+devices = pytest.helpers.mimic()
+
+light1 = devices.add("light1")(
+    "d073d5000001",
+    Products.LCM3_A19_CLEAN,
+    hp.Firmware(3, 70),
+    value_store=dict(
+        power=0,
+        color=hp.Color(0, 1, 0.3, 2500),
+    ),
+)
+
+light2 = devices.add("light2")(
+    "d073d5000002",
+    Products.LCM3_A19_CLEAN,
+    hp.Firmware(3, 70),
+    value_store=dict(
+        power=65535,
+        indication=True,
+        color=hp.Color(100, 1, 0.5, 2500),
+    ),
+)
+
+light3 = devices.add("light3")(
+    "d073d5000003",
+    Products.LCM2_A19,
+    hp.Firmware(2, 80),
+    value_store=dict(
+        power=65535,
+        color=hp.Color(100, 0, 0.8, 2500),
+    ),
+)
+
+
+@pytest.fixture(autouse=True)
+def set_async_timeout(request):
+    request.applymarker(pytest.mark.async_timeout(3))
```

Comment:
I have some changes I'm gonna commit next week when my laptop is setup that will make it so all of photons_control tests uses fake time and this won't be necessary, but for now this is fine.

basically, there are two fixtures that are used every so often FakeTime and MockedCallLater

```
it "does something", FakeTime, MockedCallLater:
    with FakeTime() as t:
        async with MockedCallLater(t):
           # Doing sleep or anything on the event loop doesn't take real time
```

I have a bunch of changes so that it's used everywhere in these tests without having to indent everything two levels.